File: Modul-4/Python/PRAK402-2310817320007-ZahraNabila.py
BatasBil = int(input("Masukkan Maksimal Batas Bilangan  : "))

print("Urutan Bilangan Ganjil            : ", end="")
for i in range(1, BatasBil + 1,):
    if i % 2 != 0:
        print(i, end=" ")

print("\nUrutan Bilangan Genap Terbalik    : ", end="")
for i in range(BatasBil, 1, -1):
    if i % 2 ==0:
        print(i, end=" ")

File: Modul-4/C/PRAK402-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
 {
    int BatasBil;

    printf("Masukkan Maksimal Batas Bilangan    : ");
    scanf("%d", &BatasBil);

    printf("Urutan Bilangan Ganjil              : ");
    for ( int i = 1; i <= BatasBil; i++)
        {
            if (i % 2 != 0)
            printf("%d ", i);
        }
    
    printf("\nUrutan Bilangan Genap Terbalik      : ");
     for (int i = BatasBil; i >= 1; i--)
    {
        if (i % 2 == 0)
        printf("%d ", i);
    }
    return 0;
}
